doc/acceleration-support.md

```diff
@@ -0,0 +1,117 @@
+# Acceleration Support
+
+The `ggml-sys` crate includes various acceleration backends, selectable via `--features` flags. The availability of supported backends varies by platform, and `ggml-sys` can only be built with a single active acceleration backend at a time. If cublas and clblast are both specified, cublas is prioritized and clblast is ignored.
+
+| Platform/OS | `cublas`           | `clblast`          | `metal`            |
+| ----------- | ------------------ | ------------------ | ------------------ |
+| Windows     | :heavy_check_mark: | :heavy_check_mark: | :x:                |
+| Linux       | :heavy_check_mark: | :heavy_check_mark: | :x:                |
+| MacOS       | :x:                | :x:                | :heavy_check_mark: |
+
+## Utilizing GPU Support
+
+To activate GPU support, set the `use_gpu` attribute of the `ModelParameters` to `true`. 
```

Comment:
"To activate GPU support (assuming that you have enabled one of the features above),"